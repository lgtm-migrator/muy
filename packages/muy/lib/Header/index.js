import React from "react";
import Box from "@material-ui/core/Box";

const Header = (props) => <Box component="header" {...props} />

export default Header
