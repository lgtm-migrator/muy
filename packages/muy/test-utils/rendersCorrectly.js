import renderer from "react-test-renderer"

const rendersCorrectly = component => {
  const tree = renderer.create(component).toJSON()
  return expect(tree).toMatchSnapshot()
}

export default rendersCorrectly
