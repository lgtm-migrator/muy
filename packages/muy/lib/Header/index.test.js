import React from "react"
import rendersCorrectly from "../../test-utils/rendersCorrectly"
import Header from "."

describe("<Header /> field", () => {
  it("should render correctly", () => {
    rendersCorrectly(<Header />)
  })
})
