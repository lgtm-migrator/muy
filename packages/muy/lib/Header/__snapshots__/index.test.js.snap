// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> field should render correctly 1`] = `
<header
  className="MuiBox-root MuiBox-root-1"
/>
`;
